docs/application/dotnet/guides/nui/nui-components/CheckBox.md

```diff
@@ -0,0 +1,78 @@
+# CheckBox
```

Comment:
Would you please check application/dotnet/guides/nui/ui-components.md?

The UI Component pages are under this dotnet/guides/nui/ui-components.md,
which is working as the cover page for the UI Components.

Please check the table in ui-components.md,
and add descriptions and links for CheckBox and RadioButton as new UI Components are added.

Reply:
Added